### ISSUE TITLE
[Bug Report] Tabs slider is not at the right position after initialization

### ISSUE THREAD

**Issue by zhangjk95** [vuetifyjs/vuetify#3233]
### Versions and Environment
**Vuetify:** 0.17.7
**Vue:** 2.5.3
**Browsers:** Chrome 64.0.3282.140
**OS:** Windows 10

### Steps to reproduce
npm install
npm run start

### Expected Behavior
The slider is under "ITEM 1".

### Actual Behavior
The slider is very wide and covers almost the whole screen.

### Reproduction Link
[https://github.com/zhangjk95/v-tabs-slider-issue](https://github.com/zhangjk95/v-tabs-slider-issue)

### Other comments
This bug does not always occur. It is more likely to occur when the web page loads slowly. I tested on my computer and it occurs very frequently.

Environment:
Processor: Intel Core i5-6300U
Memory: 8GB

**Comment by chewy94** [comment]
Looks like your repo is just using the code from the docs example and I am also unable to reproduce.

Also, since 1.0.0 we have refactored `v-tabs` and I cannot produce anything like this issue there either.

See if you can give us a better reproduction environment/case

**Comment by zhangjk95** [comment]
My point was that the problem occurs when using v-tabs with vue template and webpack. If I put the code into codepen, there will be no problem. 
Besides, the problem does not always occur. If you don't see the problem when you compile it and run it for the first time, you may try refreshing the page and you will probably see the problem.

I updated my Github repo to use the latest version of vuetify. The problem still exists, and it becomes more severe. In addition to the slider issue, the tab names do not show correctly until I click "next tab" in the example.

**Comment by chewy94** [comment]
One big thing that I notice is that you are missing `v-app` which provides much of the spacing and container functionality to Vuetify. I have generated a new project and copied your markup into it along with the data and I do not get the issue.

I did clone and run your repo and I do see the issue you described. I think this is happening because there are missing pieces in the repo to use Vuetify correctly.

I recommend using the vue-cli and generating a project and testing your code in there and see if you can reproduce.

Normally when we ask for a codepen based on our template, it is to help eliminate environment issues. When you say it works in a codepen that is more proof that this is more than likely something wrong with the reproduction repo you have provided.

Let me know if you do create a codepen or generate a project using vue-cli

**Comment by zhangjk95** [comment]
I did create my project using vue-cli, but I did not initialize the project with Vuetify. Instead, I followed the steps of integrating Vuetify into existing applications in [https://vuetifyjs.com/releases/0.17/#/vuetify/quick-start](url). In the step of including the Vuetify css file, there was a pair of **extraneous parenthesis** around the path: `import('path/to/node_modules/vuetify/dist/vuetify.min.css') // Ensure you are using css-loader` that actually caused the bug!

In the latest version of Vuetify documentation, the parentheses are gone: `import 'vuetify/dist/vuetify.min.css' // Ensure you are using css-loader`. So I tried to remove the parentheses in my repo. Then everything worked correctly!

It seems that it is a problem of the old version of the documentation, and it has been fixed now. Anyway, thanks for your response.
